pages/phases/job-analysis/creating-assessments-from-competencies.md

```diff
@@ -1,19 +1,31 @@
 ---
 layout: article
-permalink: /hiring-phases/job-analysis/creating-a-structured-interview-from-competencies/
+permalink: /hiring-phases/job-analysis/creating-assessments-from-competencies/
 section: hiring-phases
-title: Creating a Structured Interview from Competencies
+title: Creating Assessments from Competencies
 sidenav: hiring-phases
 phase: job-analysis
 sub-phase-order: 060
-intro: Finish your workshop by breaking participants into groups to write questions for both the first and second phone assessment interviews. Use your defined competencies, proficiencies, and required experience to create two rounds of interview questions and good example responses.
+intro: Finish your workshop by breaking participants into groups to write questions for both the first and second assessment rounds. Use your defined competencies, proficiencies, and required experience to create guidance for SMEs reviewing applicant responses.
 ---
 
 <p class="usa-intro">
   {{ page.intro }}
 </p>
 
-As a best practice, applicants should pass through multiple hurdles with multiple SMEs before an agency considers them qualified. Two rounds of structured phone interviews, each conducted by a single SME, assess the breadth and then the depth of applicants' knowledge about your required competencies. Any applicant who doesn't receive a high enough score during the first interview will not progress to the second.
+As a best practice, applicants should pass through multiple hurdles with multiple SMEs before an agency considers them qualified. Two rounds of assessments can assess the breadth and then the depth of applicants' knowledge about your required competencies. Any applicant who doesn't receive a high enough score during the first assessment will not progress to the second.
+
+### Assessment Options
+
+- Portfolio review
```

Comment:
the DE handbook has a list of assessment hurdles- maybe we cut and paste and reference that rather than establishing our own list (this sort of thing pisses of OPM)

Reply:
also portfolio review should be part of the initial application phase as a work sample not assessment hurdle.

Reply:
This actually has been on the site since January in another format from this PR: https://github.com/usds/SME-QA-Website/pull/107/files

I checked the [DE handbook](https://www.opm.gov/policy-data-oversight/hiring-information/competitive-hiring/deo_handbook.pdf) and it says on  2-18:

Examples of assessment tools are:
• Structured Interview
• Test
• Assessment Center
• Work Sample
• Occupational Questionnaire (Crediting Plan/Rating Schedule) 

But there's a link that took me to [this page](https://www.opm.gov/policy-data-oversight/assessment-and-selection/other-assessment-methods/) with this list:

- Accomplishment Records
- Assessment Centers
- Biodata
- Cognitive Ability
- Emotional Intelligence
- Integrity/Honesty Tests
- Job Knowledge Tests
- Personality Tests
- Reference Checking
- Situational Judgment Test
- Structured Interviews
- Training and Experience
- Work Samples

I'm going to edit that down on our site to this:
- Biodata
- Cognitive Ability
- Emotional Intelligence
- Job Knowledge Tests
- Situational Judgment Test
- Structured Interviews
- Training and Experience
- Work Samples

Saved in this commit: 
https://github.com/usds/SME-QA-Website/pull/243/commits/2e937034cb0b3272f72b69895979ca443b33a8c1